code/datums/mutations/radioactive.dm

```diff
@@ -0,0 +1,17 @@
+/datum/mutation/human/radioactive
+	name = "Radioactivity"
+	desc = "A volatile mutation that causes the host to sent out deadly beta radiation. This affects both the hosts and his surroundings."
```

Comment:
the description should not use "his", try "their" or "his or her"

Reply:
I was planning to rewrite them at some point since they were done in a pinch, I'll go ahead and change that when I do